src/components/Dialogs/Dialog/Dialog.tsx

```diff
@@ -18,42 +21,66 @@ export interface DialogProps extends ModalProps {
     /** Dialog title */
     title?: string;
     /** Properties of the accept button */
-    acceptButton: ButtonProps;
+    acceptButton?: ButtonProps;
     /** Properties of the cancel button */
     cancelButton?: ButtonProps;
+    /** the context of the buttons in the footer */
+    variant?: Omit<ButtonContext, 'secondary'>;
+    /** closes the dialog */
+    onClose: () => void;
 }
 
 const { block, elem } = bem('Dialog', styles);
 
 export const Dialog: React.FC<DialogProps> = (props) => {
-    const { isOpen, children, title, acceptButton, cancelButton, ...rest } = props;
+    const {
+        isOpen,
+        children,
+        title,
+        acceptButton,
+        cancelButton,
+        variant = 'primary',
+        onClose,
+        ...rest
+    } = props;
 
+    const { ref, height } = useResizeObserver<HTMLDivElement>();
+    const showButtons = !!acceptButton;
```

Comment:
Looking at dialog figma variations the showButtons should consider both. There is no variation with only "confirm" button. Check with designers, if it is still needed for Alert.